src/runtime_src/core/tools/xbutil2/OO_MemWrite.cpp

```diff
@@ -18,6 +18,7 @@
 #include "OO_MemWrite.h"
```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.